app/helpers/ems_physical_infra_helper/textual_summary.rb

```diff
@@ -0,0 +1,192 @@
+module EmsPhysicalInfraHelper::TextualSummary
+  include TextualMixins::TextualRefreshStatus
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i(hostname ipaddress type port cpu_resources memory_resources cpus cpu_cores guid)
+    )
+  end
+
+  def textual_group_relationships
+    TextualGroup.new(
+      _("Relationships"),
+      %i(physical_infrastructure_folders folders clusters hosts datastores vms templates orchestration_stacks ems_cloud)
+    )
+  end
+
+  def textual_group_status
+    TextualGroup.new(
+      _("Status"),
+      textual_authentications(@record.authentication_userid_passwords) + %i(refresh_status orchestration_stacks_status)
+    )
+  end
+
+  def textual_group_smart_management
+    TextualTags.new(_("Smart Management"), %i(zone tags))
+  end
+
+  def textual_group_topology
+    items = %w(topology)
+    i = items.collect { |m| send("textual_#{m}") }.flatten.compact
+    TextualGroup.new(_("Overview"), i)
+  end
+
+  #
+  # Items
+  #
+
+  def textual_hostname
+    @record.hostname
+  end
+
+  def textual_ipaddress
+    {:label => _("Discovered IP Address"), :value => @record.ipaddress}
+  end
+
+  def textual_type
+    {:label => _("Type"), :value => @record.emstype_description}
+  end
+
+  def textual_port
+    @record.supports_port? ? {:label => _("API Port"), :value => @record.port} : nil
+  end
+
+  def textual_cpu_resources
+    {:label => _("Aggregate %{title} CPU Resources") % {:title => title_for_host},
+     :value => mhz_to_human_size(@record.aggregate_cpu_speed)}
+  end
+
+  def textual_memory_resources
+    {:label => _("Aggregate %{title} Memory") % {:title => title_for_host},
+     :value => number_to_human_size(@record.aggregate_memory * 1.megabyte, :precision => 0)}
+  end
+
+  def textual_cpus
+    {:label => _("Aggregate %{title} CPUs") % {:title => title_for_host}, :value => @record.aggregate_physical_cpus}
+  end
+
+  def textual_cpu_cores
+    {:label => _("Aggregate %{title} CPU Cores") % {:title => title_for_host},
+     :value => @record.aggregate_cpu_total_cores}
+  end
+
+  def textual_guid
+    {:label => _("Management Engine GUID"), :value => @record.guid}
+  end
+
+  def textual_physical_infrastructure_folders
+    return nil if @record.kind_of?(ManageIQ::Providers::PhysicalInfraManager)
```

Comment:
@himdel It looks like it's copy-pasta, because it is :) Ems Infra has been a good start to base Physical Infra on, but sometimes we do have unneeded code.

Didn't realise that's what `@record` here was used for, as there weren't any comments 😌